IntegrationTests/UITests/Script/ScriptRunnerTests.cs

```diff
@@ -181,13 +180,21 @@ public void RunScript_with_arguments_with_s_option_with_RevisionGrid_with_select
             RunFormTest(formBrowse =>
             {
                 // wait until the revisions are loaded
-                while (formBrowse.RevisionGridControl.LatestSelectedRevision == null)
+                AsyncTestHelper.WaitForPendingOperations(AsyncTestHelper.UnexpectedTimeout);
```

Comment:
This `WaitForPendingOperations` times out very often.
What is different with tests which open FormBrowse?
- `RevisionDataGridView._backgroundProcessingTask` using `ThreadHelper.JoinableTaskContext.Factory`
- FormBrowse is opened using `Application.Run` because `Application.MessageLoop` is `false` when running tests
- something else?